web-app-router/tailwind.config.ts

```diff
@@ -51,14 +54,19 @@ const config: Config = {
             500: "#00B800",
           },
         },
+
+        gridTemplateColumns: {
+          "1fr/auto": "1fr auto",
+          "auto/1fr": "auto 1fr",
+          "auto/1fr/auto": "auto 1fr auto",
+        },
```

Comment:
this is not working for me btw @igorosip0v  using grid-cols-auto/1fr/auto

Reply:
it was inside color object, fixed